source/postprocess/heating_statistics.cc

```diff
@@ -114,6 +114,11 @@ namespace aspect
 
             this->get_material_model().evaluate(in, out);
 
+            if (this->get_parameters().formulation_temperature_equation
```

Comment:
Sure. I also added an else block with the other option (with an empty block) and an else ``ExcNotImplemented`` in case we ever extend the number of enum options.